Design/Instrument-2.0/Detector_Scan_NeXus_Saving.md

```diff
@@ -0,0 +1,129 @@
+Detector Scan NeXus Saving
+==========================
+
+## Background
+
+Workspaces in Mantid can now support detector scans, but these can not currently be saved in the Mantid NeXus Processed format. In related work the ESS are looking at saving a new style instrument definition in their raw files, to be read, and eventually saved by Mantid. The new style instrument should be able to save the scan information within itself, but for now something similar could be stored separately in the Mantid NeXus processed files, that will be moved when the NeXus instrument definition is available.
```

Comment:
This sounds like it would create the need to support legacy files, once we have Nexus instrument definition? Is there a way to avoid such a temporary solution?

Reply:
Ideally yes, we should avoid any legacy files. The scan saving support is not urgent, ideally it would be good to have something on the months time scale, so it might be possible to avoid new legacy files.